controllers/admin/AdminCarriersController.php

```diff
@@ -506,7 +506,7 @@ public function postProcess()
                 // Delete from the reference_id and not from the carrier id
                 $carrier = new Carrier((int) $id);
                 Warehouse::removeCarrier($carrier->id_reference);
-            } elseif (Tools::isSubmit($this->table . 'Box') && count(Tools::getValue($this->table . 'Box', [])) > 0) {
+            } elseif (Tools::isSubmit($this->table . 'Box') && count(Tools::isSubmit($this->table . 'Box')) > 0) {
```

Comment:
Not count on `Tools::isSubmit` which returns a boolean.

Reply:
And you would be reverting a recent fix https://github.com/PrestaShop/PrestaShop/pull/20960 🤔

Reply:
Yes, this isn't my change and this is totally not related with this PR

Reply:
@ks129 can you revert this change please?